unit-tests/test-fw-update.py

```diff
@@ -72,6 +74,27 @@ def reset_update_counter( device ):
 
     send_hardware_monitor_command( device, cmd )
 
+def find_image_or_exit( product_name, image_mask ):
+    pattern = re.compile('Intel RealSense ')
+    match = pattern.search(product_name)
+    if not match:
+        # Didn't find 'Intel RealSense ' in product_name
```

Comment:
I'd say put this comment in the error: you need the error to explain what isn't valid...

Reply:
done